openstack/nat/v3/gateways/requests.go

```diff
@@ -33,7 +33,7 @@ type CreateOpts struct {
 }
 
 // DownLinkVpc is an object that represents the subnet configuration to which private NAT gateway belongs.
-type DownLinkVpc struct {
+type DownLinkVpcOpt struct {
```

Comment:
```suggestion
type DownLinkVpc struct {
```
changing this name will cause code incompatibility.

Reply:
done